.github/workflows/build-macos.yml

```
@@ -19,16 +19,65 @@ jobs:
         working-directory: src
 
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)